setup.py

```diff
@@ -15,7 +15,7 @@
         'masonite.contracts',
         'masonite.helpers',
     ],
-    version='1.6.5',
+    version='1.6.4',
```

Comment:
This should be 1.6.6 so we can release the next bug fix

Reply:
Oh shoot! Sorry! I'll get that done.